fixtures/room/room_analytics_duration_fixtures.py

```diff
@@ -0,0 +1,97 @@
+null = None
+
+get_daily_meetings_total_duration_query = '''
+query {
+    dailyDurationsOfMeetings(locationId:1, dayStart:"sep 10 2018"){
+        MeetingsDurationaAnalytics{
+            roomName
+            count
+            totalDuration
+            events{
+                durationInMinutes
+                numberOfMeetings
+            }
+        }
+    }
+}
+'''
+
+get_daily_meetings_total_duration_response = {
+    "data": {
+        "dailyDurationsOfMeetings": {
+            "MeetingsDurationaAnalytics": [
+                {
+                    "roomName": "Entebbe",
+                    "count": 0,
+                    "totalDuration": 0,
+                    "events": []
+                }
+            ]
+        }
+    }
+}
+
+get_weekly_meetings_total_duration_query = '''
+query {
+    weeklyDurationsOfMeetings(locationId:1, weekStart:"Jan 3 2018", weekEnd:"Jan 9 2021"){  # noqa: E501
```

Comment:
I see here that the week End date is Jan 9 2021. Is this a typo?

Reply:
@3V0L . Thank you for identifying this. Solved already